packages/devtools_app/lib/src/screens/debugger/controls.dart

```diff
@@ -90,29 +90,32 @@ class _DebuggingControlsState extends State<DebuggingControls>
     required bool resuming,
   }) {
     final isSystemIsolate = controller.isSystemIsolate;
+    // Note: this could be refactored to use a single [GaDevToolsButton] instead
+    // of a [RoundedButtonGroup] with only one child, but this would require
+    // changes to the underlying [DevToolsButton] widget from
+    // package:devtools_app_shared to accept an `iconAsset` as a parameter.
```

Comment:
Is this worth filing an issue for?

Reply:
I left a note instead of a TODO per our discussion at the DevExp kick off meeting. I think this would be an issue filed that never becomes important enough to address, so I opted to leave a note instead of filing an issue since addressing this is unimportant.